packages/nextjs/test/run-integration-tests.sh

```diff
@@ -78,6 +78,19 @@ for NEXTJS_VERSION in 10 11 12; do
       WEBPACK_VERSION=5 ||
       WEBPACK_VERSION=4
 
+    # Node v18 only with Webpack 5 and above
+    # https://github.com/webpack/webpack/issues/14532#issuecomment-947513562
+    # Context: https://github.com/vercel/next.js/issues/30078#issuecomment-947338268
+    if [ "$NODE_MAJOR" -gt "17" ] && [ "$WEBPACK_VERSION" -eq "4" ]; then
+      echo "[nextjs$NEXTJS_VERSION | webpack@$WEBPACK_VERSION] Node $NODE_MAJOR not compatible with Webpack $WEBPACK_VERSION"
+      exit 0
+    fi
+    if [ "$NODE_MAJOR" -gt "17" ] && [ "$NEXTJS_VERSION" -eq "10" ]; then
+      echo "[nextjs$NEXTJS_VERSION | webpack@$WEBPACK_VERSION] Node $NODE_MAJOR not compatible with Webpack $WEBPACK_VERSION"
```

Comment:
Not sure about this one. Did you intend it this way?

```suggestion
      echo "[nextjs@$NEXTJS_VERSION | webpack@$WEBPACK_VERSION] Node $NODE_MAJOR not compatible with next.js $NEXTJS_VERSION"
```

Reply:
Nope, thanks for catching that!